mayastor/src/subsys/config/mod.rs

```diff
@@ -286,7 +286,10 @@ impl Config {
         let pools = PoolsIter::new()
             .map(|p| Pool {
                 name: p.get_name().into(),
-                disks: vec![p.get_base_bdev().name()],
+                disks: vec![p
+                    .get_base_bdev()
+                    .bdev_uri()
+                    .unwrap_or_else(|| p.get_base_bdev().name())],
                 blk_size: p.get_base_bdev().block_len(),
```

Comment:
is it worth abstracting out the repeated calls to get_base_bdev()
ie.
`.map(|p| { let base = p.get_base_bdev() ; Pool { ... } }`

Reply:
I avoided it because it makes the change a bit more "verbose" but maybe it's better. I've pushed it.